pysmt/oracles.py

```diff
@@ -109,6 +115,12 @@ def walk_count_symbols(self, formula, args, measure, **kwargs):
             return frozenset([formula]) | a_res
         return a_res
 
+    def walk_count_bool_dag(self, formula, args, measure, **kwargs):
+        #pylint: disable=unused-argument
+        if formula.is_bool_op():
+            return frozenset([formula]) | frozenset([x for s in args for x in s])
```

Comment:
In this way, you are only counting the Boolean connectives, right?

Boolean variables, an theory atoms count as 0, is this right?